README.md

```diff
@@ -3,6 +3,9 @@ Automatically Fill Cycle in Lab Evaluation Sheet on the back side of Index page.
 
 Website - <a href="http://fillmycycle.ga" target="_blank">http://fillmycycle.ga</a>
```

Comment:
remove this link.

alternatively, i would advise you to replace the older non-functional URL to the new one

Reply:
i did merge your PR, but after reviewing again, realized that you haven't removed the older link. I have reverted that merge. Kindly do it the way I have suggested in the above comment

Reply:
follow this syntax: [link](link)